rs/nns/integration_tests/src/bad_input.rs

```diff
@@ -34,7 +34,7 @@ fn test_skipping_quota() {
             .query_("get_pending_proposals", candid, (skipped,))
             .await;
         let err = res.unwrap_err();
-        let expected_err = "Deserialization Failed";
```

Comment:
Why?

Reply:
dfn_core -> ic_cdk. dfn_core defined part of the error messages